iOS/UI/Manga/MangaViewController.swift

```diff
@@ -1160,6 +1160,21 @@ extension MangaViewController: MangaDetailHeaderViewDelegate {
         present(SFSafariViewController(url: url), animated: true)
     }
 
+    // copy manga link when holding down the web view button
+    func safariHeld() {
+        guard
+            let url = manga.url,
+            url.scheme == "https" || url.scheme == "http"
```

Comment:
does the url scheme matter when copying it? it shouldn't be anything other than these two things in normal cases, but I don't think there's any reason to actually check it since if it were different it wouldn't really matter.